spec/adapters/alma_adapter_spec.rb

```diff
@@ -306,38 +216,13 @@
             }
           )
           .to_return(status: 200, body: "{}", headers: { "content-Type" => "application/json" })
-
```

Comment:
We should remove the "no holding notes" part from the spec title here. And can the first stub also be removed? It was added in https://github.com/pulibrary/marc_liberation/pull/1052

Reply:
Removed.